### PR TITLE
HTML/CSS & JavaScript capstone project 

### PR DESCRIPTION
**Pull request by alphym**
<h3>HTML/CSS & JavaScript capstone project </h3>

-  I personalized the content of my page. Choose a topic that is a Techgirl meetup.

- I built only these 2 pages:

- The home page.
- The about page.
- Each of these pages has versions for 2 different screen sizes:

- Mobile: up to 768px wide.
- Desktop: 768px or wider.
- Interactions

link to video  project presentation
[https://www.loom.com/share/08cbf0723f6e424ebcd9d05a015fd8bd](url)

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+                <li class="mobile-li"><a href=""></a>Join</li>
+                <li class="mobile-li"><a href=""></a>Sponsor</li>
+                <li class="mobile-li"><a href=""></a>News</li>
+                <li class="mobile-li"><a href=""></a> contact</li>
+        
+            </ul>
+        </div>
+
+        <div class="main-content">
+            <h2 class="header-title">GIRLS IN TECH MEETUP 2023</h2>
```
Comment (GedeonTS) [inline about.html]: - [x] In order to respect the code semantic HTML rules, I suggest using h2 only if you already have h1 on the page, Kindly fix it as I don't see it before :recycle:

File: style.css [hunk 2]
```diff
@@ -0,0 +1,721 @@
...
+    padding-left: 150px;
+    padding-right: 150px;
+    gap: 10px;
+  }
+
+  .partners .partners-title {
+    width: 100%;
+    align-items: center;
+
+    /* background-color: white; */
```
Comment (GedeonTS) [inline style.css]: - [  ] Kindly make sure to remove all commented-out lines of code to make your code cleaner :recycle:

File: style.css [hunk 3]
```diff
@@ -0,0 +1,721 @@
...
+  .partners-title hr {
+    width: 20px;
+    border: 0.5px solid #8d4940;
+  }
+
+  .partners-list {
+    width: 100%;
+    height: 70%;
+
+    /* background-color: #f45917; */
```
Comment (GedeonTS) [inline style.css]: - [x] Kindly remove this line too in order to make your code cleaner :recycle:

File: script.js [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+                <div class="card-feature-img">
+                    <img src="${featureSpeakers[i].img}" alt="">
+                </div>
+                <div class="card-feature-description">
+                    <h2 class="header-two">${featureSpeakers[i].name}</h3>
+                        <h3 class="header-three">${featureSpeakers[i].title}</h3>
+                        <hr>
+                        <p class="par-card"> ${featureSpeakers[i].description}</p>
+                </div>
+            </div>`;
```
Comment (GedeonTS) [inline script.js:70]: - [x] Kindly make sure this section looks like the Behance design, below are some hints:
  - Adding the grey chess
Screenshot from 2023-02-18 00-40-11

  - removing the border

File: style.css [hunk 5]
```diff
@@ -0,0 +1,721 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,300;0,400;0,700;0,900;1,300&display=swap');
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+
+  /* padding: 0; */
```
Comment (GedeonTS) [inline style.css]: - [x] Kindly make sure to delete this line too and all other commented-out lines of code to make your code clean :recycle: :

File: index.html [hunk 6]
```diff
@@ -0,0 +1,187 @@
...
+            <div class="copyright">
+               <p class="par">"Tech girls meetup (2023),"
+                
+                "All right Reserved."
+               </p>
+              
+            </div>
+        </div>
+        
+    </footer>
```
Comment (GedeonTS) [inline index.html:180]: - [ ] Kindly make sure this section doesn't appear on the mobile version as it is on Cindy's design.

Screenshot from 2023-02-18 01-02-17

File: index.html [hunk 7]
```diff
@@ -0,0 +1,187 @@
...
+                <li class="mobile-li"><a href=""></a>Join</li>
+                <li class="mobile-li"><a href=""></a>Sponsor</li>
+                <li class="mobile-li"><a href=""></a>News</li>
+                <li class="mobile-li"><a href=""></a> contact</li>
+            
+            </ul>
+        </div>
+        <div class="main-content">
+            <q class="quote">hello girls in TECH</q>
+            <h1 class="header-title">GIRLS IN TECH MEETUP 2023</h1>
```
Comment (GedeonTS) [inline index.html:54]: - [ ] Kindly make sure to increase your font size and use the right color for this title as it is on Cindy's design :recycle: 

- Your design :
Screenshot from 2023-02-18 01-04-01

- Behance:
Screenshot from 2023-02-18 01-06-49

File: index.html [hunk 8]
```diff
@@ -0,0 +1,187 @@
...
+            <ul class="mobile-ul" >
+                <img id="close-btn" src="./images/close.png" alt="">
+                <li class="mobile-li"><a class="mobile-about" href="./about.html">About</a></li>
+                <li class="mobile-li"><a href=""></a>Program</li>
+                <li class="mobile-li"><a href=""></a>Join</li>
+                <li class="mobile-li"><a href=""></a>Sponsor</li>
+                <li class="mobile-li"><a href=""></a>News</li>
+                <li class="mobile-li"><a href=""></a> contact</li>
+            
+            </ul>
```
Comment (GedeonTS) [inline index.html]: - [ ] Kindly make the background of your headline and the header are the same to stick to Cindy's design

Screenshot from 2023-02-18 01-11-41

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED :recycle: :

Hi @alphym , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights :tophat:  :+1: 
- No linter errors :+1: 
- Good use of GitHub flow :+1: 
- Amazingly responsive design :clap: 
## Required Changes ♻️
- [ ] Good job on recording a descriptive video for the project :clap:  However, :sob: : you forgot to show us a piece of code that you liked the most, kindly to meet that requirement :recycle: 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by m05e5 — APPROVED** [review]
**Hi @alphym 
You did great. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
CleverScornfulBuffalo-size_restricted

## Highlights

✔️ Nice UI
✔️ No linters error

## Optional suggestions



_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**m05e5**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._**

**Pull request merged**